app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -209,70 +261,43 @@ class RecentlyPlayedFragmentTest {
           R.id.lesson_thumbnail
         )
       ).perform(click())
-
       intended(
         allOf(
-          hasExtra(ExplorationActivity.EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY, FRACTIONS_EXPLORATION_ID_1),
+          hasExtra(ExplorationActivity.EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY, FRACTIONS_EXPLORATION_ID_0),
+          hasExtra(ExplorationActivity.EXPLORATION_ACTIVITY_STORY_ID_ARGUMENT_KEY, FRACTIONS_STORY_ID_0),
           hasExtra(ExplorationActivity.EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY, FRACTIONS_TOPIC_ID),
+          hasExtra(ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY, internalProfileId),
           hasComponent(ExplorationActivity::class.java.name)
         )
       )
     }
   }
 
   @Test
-  @Ignore("Only one item available from domain layer") // TODO(#77): Reenable this test
-  fun testRecentlyPlayedTestActivity_recyclerViewItem3_showsLastMonthSectionTitle() {
-    ActivityScenario.launch(RecentlyPlayedFragmentTestActivity::class.java).use {
-      onView(withId(R.id.ongoing_story_recycler_view)).perform(scrollToPosition<RecyclerView.ViewHolder>(3))
+  fun testRecentlyPlayedTestActivity_recyclerViewItem2_showsLastMonthSectionTitle() {
+    ActivityScenario.launch<RecentlyPlayedActivity>(createRecentlyPlayedActivityIntent(internalProfileId)).use {
+      onView(withId(R.id.ongoing_story_recycler_view)).perform(scrollToPosition<RecyclerView.ViewHolder>(2))
       onView(
         atPositionOnView(
           R.id.ongoing_story_recycler_view,
-          3,
+          2,
           R.id.section_title_text_view
         )
```

Comment:
Here also Indentation is incorrect. `atPositionOnView` method can be written in single line. Please correct it here and also at other similar places in this file.

Reply:
Indentation is something which we cannot confirm that it should follow  one single pattern because of android studio. The only thing that we can make sure is that the indentation in one single file is consistent. I have done that. Thanks.